### PR TITLE
ENT-10888: Added changelog for 3.23.0

### PR DESCRIPTION
**Pull request by craigcomstock**

### CODE REVIEW

File: ChangeLog [hunk 1]
```diff
@@ -1,3 +1,51 @@
...
+	  If the default software inventory from context is not specified, the
+	  package[update]smatching policy functions now looks for the software
+	  inventory databases in the state directory and uses them if found. This
+	  change enables the usage of these functions in standalone policy files
+	  without the demand for specifying the default package inventory
+	  attribute in body common control. However, you still need the default
+	  package inventory attribute specified in the policy framework for the
+	  software inventory databases to exist in the first place and to be
+	  maintained.
+	  Ticket: (ENT-9083)
```
Comment (larsewi) [inline ChangeLog:47]:
```suggestion
	- packagesmatching() and packageupdatesmatching() now look for the software
	  inventory databases in the state directory and use them if found. This
	  change enables the usage of these functions in standalone policy files
	  without the demand for specifying the default package inventory attribute
	  in body common control. However, you still need the default package
	  inventory attribute specified in the policy framework for the software
	  inventory databases to exist in the first place and to be maintained.
	  (ENT-9083)
```

File: ChangeLog [hunk 2]
```diff
@@ -1,3 +1,51 @@
+3.23.0:
+	- Added selinux policy to allow cf-hub to initiate scheduled reports
+	  (ENT-10696, ENT-9825)
+	- Added version_compare() policy function (CFE-3991)
+	- Bodies can now inherit attributes containing global variables
+	  (CFE-4254)
+	- Cashed policy function results now take into account number of arguments and function name.
+	  Co-authored-by: Alexis Mousset <alexis.mousset@rudder.io> (CFE-4244)
```
Comment (larsewi) [inline ChangeLog:8]:
```suggestion
	- Cached policy function results now take into account number of arguments
	  and function name (CFE-4244)
```

File: ChangeLog [hunk 3]
```diff
@@ -1,3 +1,51 @@
...
+	- Corrected CFEngine capitalization
+	- Fixed inconsistent use of tabs and spaces
```
Comment (larsewi) [inline ChangeLog:10]:
```suggestion
```

File: ChangeLog [hunk 4]
```diff
@@ -1,3 +1,51 @@
...
+	  (ENT-9985)
+	- Modified classesmatching() function to search parent bundles with inherit => true
+	  (ENT-5850)
+	- Moved ignore_interfaces.rx to $(sys.workdir)
+	  Moved expected location of ignore_interfaces.rx from $(sys.inputdir) to
+	  $(sys.workdir). If the file is found in $(sys.inputdir) but not in
+	  $(sys.workdir), we will still process it for backwards compatability,
+	  but issue a warning prompting the user to move it to the appropriate
+	  location.
+	  Ticket: (ENT-9402)
```
Comment (larsewi) [inline ChangeLog:26]:
```suggestion
	- Moved expected location of ignore_interfaces.rx from $(sys.inputdir) to
	  $(sys.workdir). If the file is found in $(sys.inputdir) but not in
	  $(sys.workdir), we will still process it for backwards compatability,
	  but issue a warning prompting the user to move it to the appropriate
	  location. (ENT-9402)
```

File: ChangeLog [hunk 5]
```diff
@@ -1,3 +1,51 @@
...
+	- Only CFEngine processes are now killed as expired lock owners
+	  (CFE-3982)
+	- Removed unnecessary newlines
```
Comment (larsewi) [inline ChangeLog:29]:
```suggestion
```

File: ChangeLog [hunk 6]
```diff
@@ -1,3 +1,51 @@
...
+	- SELinux no longer blocks CFEngine deamons in reading security parameters from /proc/sys/kernel
+	  (ENT-9684)
+	- Updated old changelog entries to match more recent ones
```
Comment (larsewi) [inline ChangeLog:32]:
```suggestion
```

File: ChangeLog [hunk 7]
```diff
@@ -1,3 +1,51 @@
...
+	- cf-hub is now allowed to use the TLS kernel module on
+	             SELinux-enabled systems (ENT-9727)
+	- cf_lock.lmdb is no longer restored from backup on
+	             every boot (CFE-3982)
```
Comment (larsewi) [inline ChangeLog:36]:
```suggestion
	- cf-hub is now allowed to use the TLS kernel module on
	  SELinux-enabled systems (ENT-9727)
	- cf_lock.lmdb is no longer restored from backup on
	  every boot (CFE-3982)
```

**Review by larsewi — CHANGES_REQUESTED** [review]

File: ChangeLog (hunk 3 above)
Comment (nickanderson) [reply ChangeLog:10]: fing tabs

**Review by nickanderson — COMMENTED** [review]
Lars suggestions look good to me.

**Pull request merged**